components/EmailAddressRow/index.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
```

Comment:
This is a separate component for each email row in the `Email and Authentication` section of the profile page.